src/System.Collections/src/Resources/Strings.resx

```diff
@@ -202,7 +193,7 @@
     <value>Target array type is not compatible with the type of items in the collection.</value>
   </data>
   <data name="ArgumentOutOfRange_BiggerThanCollection" xml:space="preserve">
-    <value>Larger than collection size.</value>
+    <value>Must be less than or equal to the size of the collection.</value>
```

Comment:
Neither the old nor the new wording seems to make sense here:
https://github.com/dotnet/corefx/pull/5672/files#diff-78d6503f5065af177a325d9668fe4735L903
or here:
https://github.com/dotnet/corefx/pull/5672/files#diff-78d6503f5065af177a325d9668fe4735R904